src/mmap.cpp

```diff
@@ -49,22 +49,22 @@ mmap_reads(off_t partial_file_size,
            int32_t &offset_pair_from_top,
```

Comment:
mmap is limited by the virtual address space, so should be 2**32 bytes for a 32-bit system and 2**64 for a 64-bit